src/goose/console/views/redis/home.clj

```diff
@@ -0,0 +1,30 @@
+(ns goose.console.views.redis.home
+  (:require [goose.console.views.layout :as layout]))
+
+(defn header [_]
+  [:header
+   [:nav
+    [:div.nav-start
+     [:div.goose-logo
+      [:a {:href "/"}
+       [:img {:src "img/goose-logo.png" :alt "goose-logo"}]]]
+     [:div#app-name "AppName"]
```

Comment:
question: \<Appname\> is currently static name in the navbar. It will be made dynamic(based on user input) in another story. Let me know you want to address it in this story itself @olttwa